decidim-bulletin_board-app/app/commands/process_key_ceremony_step.rb

```diff
@@ -44,10 +44,10 @@ def call
 
   private
 
-  attr_accessor :trustee, :response_log_entry
+  attr_accessor :trustee, :response_log_entries
 
   def save_election!
-    if response_log_entry && response_log_entry.message_identifier.type == "end_key_ceremony"
+    if response_log_entries && response_log_entries.any? { |le| le.message_identifier.type == "end_key_ceremony" }
```

Comment:
I guess that here and in the tally we can assume that the `end_...` will be the last response, right? Even with that, maybe this approach is healthier. 🤔